services/workflows-service/prisma/schema.prisma

```diff
@@ -222,6 +222,7 @@ model File {
   mimeType         String?
   userId           String
   fileNameOnDisk   String
+  fileName         String?
```

Comment:
@Omri-Levy Create a data script that set file name for all the other rows in files table which use the migration process @Blokh built.

```
const { PrismaClient } = require('@prisma/client');
const path = require('path');
const prisma = new PrismaClient();

async function processFiles() {
  const pageSize = 1000; // Adjust as needed
  let skip = 0;
  let totalCount = 0;

  do {
    const files = await prisma.file.findMany({
      skip,
      take: pageSize,
    });

    if (files.length === 0) {
      break; // No more records to process
    }

    for (const file of files) {
      const fileName = extractFileNameFromUri(file.uri);
      console.log(`Processing file: ${fileName}`);
      
      // Update the file record with the extracted file name
      await prisma.file.update({
        where: { id: file.id },
        data: { fileName },
      });
    }

    totalCount += files.length;
    skip += pageSize;

    console.log(`Processed ${totalCount} files so far...`);
  } while (true);
}

function extractFileNameFromUri(uri) {
  return path.basename(uri);
}

processFiles();
```

Reply:
Later change fileName column to not null

Reply:
I believe we should do this once a need arises, then allocate a time and priority.